plone/app/z3cform/widget.py

```diff
@@ -90,7 +90,14 @@ def render(self):
         """
         if self.mode != 'input':
             return super(BaseWidget, self).render()
-        return self._base(**self._base_args()).render()
+
+        pattern_widget = self._base(**self._base_args())
+        if getattr(self, 'klass', False):
+            pattern_widget.klass = u'{0}{1}'.format(
+                pattern_widget.klass,
+                u' ' + self.klass
```

Comment:
Ah true - I did not want an extra space if ``self.klass`` is None, but that's never the case with the ``if getattr`` above. tnx!